monkey_island/cc/services/config.py

```diff
@@ -217,6 +217,35 @@
                                 " Examples: \"192.168.0.1\", \"192.168.0.5-192.168.0.20\", \"192.168.0.5/24\""
                         }
                     }
+                },
+                "network_analysis": {
+                    "title": "Network Analysis",
+                    "type": "object",
+                    "properties": {
+                        "inaccessible_subnet_groups": {
+                            "title": "Inaccessible IP/subnet groups",
+                            "type": "array",
+                            "uniqueItems": True,
+                            "items": {
+                                "type": "array",
+                                "title": "Subnet group",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "minItems": 2,
+                                "uniqueItems": True,
+                                "description": "List of IPs/subnets."
+                                               " Examples: \"192.168.0.1\", \"192.168.0.5-192.168.0.20\","
```

Comment:
What you're providing is single items, not groups, not sure what the example is trying to say.

Reply:
This is how it looks like from the island. Is this unclear? What would you change?

![image](https://user-images.githubusercontent.com/30774653/40414544-29b081d0-5e81-11e8-8922-9f1b274ea03d.png)

Reply:
No. I, as a user, do not understand how this is different from the IPs to attack display.